Design/Workbench/design-layout.md

```diff
@@ -0,0 +1,149 @@
+Summary
+=======
+
+This document describes the layout and high-level details of the new matplotlib-based workbench that will replace MantidPlot.
+
+Workflow
+========
+
+The new workbench will be developed on a long-running feature branch within the main [`mantid`][mantidrepo] repository. The exception to this is the new directory
+structure described below for the graphical components. These changes will be made on the `master` branch prior to any work starting on the workbench. This
+will minimize the effect of merge conflicts on existing components that are going to be reused.
+
+The advantage here is that we can remove the existing MantidPlot on this branch and reuse the name for the new workbench.
+
+
+Directory Structure
+===================
+
+The following diagram indicates the proposed directory structure for the graphical components.
+
+```
+mantid.git
+   |-- Framework
+   |-- MantidPlot
+   |-- qt
+       |-- applications
+       |   |-- mantidworkbench
```

Comment:
I would either leave it here or do away with the `applications` directory and have it there. To me anything different requires you to learn where it is anyway.